modules/insights-operator-copying-archive.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/remote-health-reporting-from-restricted-network.adoc
+
+[id="insights-operator-copying-archive_{context}"]
+= Copying the Insights Operator archive
+
+Make a copy of your Insights Operator data archive for upload to link:https://cloud.redhat.com[cloud.redhat.com].
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Find the name of the currently running pod for the Insights Operator:
++
+[source,terminal]
+----
+$ INSIGHTS_OPERATOR_POD=$(oc get pods --namespace=openshift-insights -o custom-columns=:metadata.name --no-headers  --field-selector=status.phase=Running)
+----
+
+. Copy the recent data archives collected by the Insights Operator:
```

Comment:
Maybe we can say that it's copying from the IO container?

Reply:
how about `. Copy the recent data archives from Insights Operator container:` ?